Documentation/Releases/solr release 6.0.md

```diff
@@ -84,11 +101,13 @@ The amount of ping requests has been reduced to only do one ping when the plugin
 * https://github.com/TYPO3-Solr/ext-solr/pull/784
 * https://github.com/TYPO3-Solr/ext-solr/pull/776
 
-### Allow solr side facet limit
+### Allow Solr Server-Side Facet Option Limits
```

Comment:
@timohund I believe this was/is about the number of facet **options**, correct me if I'm wrong.